projectHelpers.js

```diff
@@ -30,11 +30,36 @@ const writePackageJson = (content, projectDir) => {
     fs.writeFileSync(packageJsonPath, JSON.stringify(content, null, 2))
 }
 
+const getProjectDir = ({ nestingLvl } = { nestingLvl: 0 }) => {
```

Comment:
This may be useful: https://www.npmjs.com/package/app-root-path or may be not. :)